convert2rhel.ini.example

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Example of configuration file convert2rhel.ini for secrets.
+# Possible locations of this file:
+# 1) user specified and passed by -c, --config-file option; highest priority
+# 2) ~/convert2rhel.ini; lower priority
+# 3) /etc/convert2rhel.ini; the lowest priority
+
+# [SECRET]
```

Comment:
Using a header. Due [1] I decide, that using it is a good practice and can provide better readability if the config file would be used for something else. If the header isn't needed there is possibility to do it without it - adding some dummy section after reading the file so the configparser accepts the structure.

[1] [https://docs.python.org/3/library/configparser.html#supported-ini-file-structure](https://docs.python.org/3/library/configparser.html#supported-ini-file-structure)